plugins/linkattacher/src/linkattacher.cc

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2013-2015 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#include <linkattacher.hh>
+
+using namespace gazebo;
+using namespace std;
+using namespace yarp::os;
+
+LinkAttacher::LinkAttacher() : ModelPlugin(), m_network(), m_rpcport()
```

Comment:
You don't need to initialize the unique pointers

Reply:
Done